app/controllers/miq_ae_customization_controller.rb

```diff
@@ -104,6 +106,14 @@ def accordion_select
     replace_right_cell(:nodetype => x_node)
   end
 
+  def self.session_key_prefix
+    "miq_ae_customization"
+  end
+
+  def title
+    "miq_ae_customization"
```

Comment:
`title` should return a human readable string - it will be shown in the tab title (maybe :))

So maybe something like `_("Automate Customization")`?